pkg/cli/admin/release/mirror.go

```diff
@@ -453,8 +453,10 @@ func (o *MirrorOptions) Run() error {
 		fmt.Fprintf(o.Out, "\nSuccess\nUpdate image:  %s\nMirrored to: %s\n", to, o.To)
 	}
 
-	if err := printImageContentInstructions(o.Out, o.From, o.To, repositories); err != nil {
-		return fmt.Errorf("Error creating mirror usage instructions: %v", err)
+	if len(o.From) > 0 && len(o.To) > 0 {
```

Comment:
The code in printimageContentInstructions is wrong, o.From is not required to be set.  So this PR should change the method to not error out if from is empty in printImageContentInstructions, and then we should add a boolean that says whether to print this thing, and set it true here and it can remain default false in new.go

Reply:
>  So this PR should change the method to not error out if from is empty...

Done with a4d060b77 -> da15a34ed.

> ... and then we should add a boolean that says whether to print this thing, and set it true here and it can remain default false in new.go.

I'm still not clear on why we want this.  If `from` is empty, what would you say?

Reply:
In the regular flow, `from` will always be set, b/c that's how mirror command works and w/o it it fails `Complete` invocation. The fact that you have the `from` being empty check is just to ensure we don't do stupid things. The boolean is to completely skip `printImageContentInstructions` invocation in here.

Reply:
I've pushed da15a34ed -> 4baf093e6 with a new `PrintImageContentInstructions`, now that I understand the non-empty-`repositories` angle.